cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/MessagingServiceMain.java

```diff
@@ -80,4 +82,9 @@ protected LoggingContext getLoggingContext(EnvironmentOptions options) {
                                      Constants.Logging.COMPONENT_NAME,
                                      Constants.Service.MESSAGING_SERVICE);
   }
+
+  @Override
+  protected void initializeDataSourceConnection(CConfiguration cConf) throws SQLException {
```

Comment:
You can remove the `throws` since this method is no-op (override method can throw less exception then the parent).

Reply:
Done